app/models/comfy/cms/page.rb

```diff
@@ -90,4 +90,14 @@ def ever_been_published?
   def publishable?
     keywords.all?(&:publishable?)
   end
+
+  def published_at
+    if published?
+      updated_at
+    elsif scheduled?
+      scheduled_on
```

Comment:
I am curious. Is this scheduled expected as a business rule/acceptance criteria?